src/main/java/com/gmail/yuramitryahin/Main.java

```diff
@@ -2,19 +2,36 @@
 
 import com.gmail.yuramitryahin.exception.AuthenticationException;
 import com.gmail.yuramitryahin.lib.Injector;
+import com.gmail.yuramitryahin.model.Movie;
+import com.gmail.yuramitryahin.model.MovieSession;
 import com.gmail.yuramitryahin.model.User;
 import com.gmail.yuramitryahin.service.AuthenticationService;
+import com.gmail.yuramitryahin.service.MovieService;
+import com.gmail.yuramitryahin.service.MovieSessionService;
+import com.gmail.yuramitryahin.service.ShoppingCartService;
 
 public class Main {
     private static Injector injector = Injector.getInstance("com.gmail.yuramitryahin");
+    private static final AuthenticationService authenticationService
+            = (AuthenticationService) injector.getInstance(AuthenticationService.class);
+    private static final ShoppingCartService shoppingCartService = (ShoppingCartService) injector
+            .getInstance(ShoppingCartService.class);
+    private static final MovieSessionService movieSessionService = (MovieSessionService) injector
+            .getInstance(MovieSessionService.class);
+    private static final MovieService movieService = (MovieService) injector
+            .getInstance(MovieService.class);
+    private static final User user = authenticationService
+            .register("user@gmail.com", "1234");
```

Comment:
let's make this a local variable

Reply:
but if I make local, check style has a problem and write that user must be final

Reply:
I made local final variable